src/pages/account/index.tsx

```diff
@@ -33,10 +34,7 @@ const getServerSideProps: GetServerSideProps = async (context) => {
     context,
     [
       tracingMiddleware(getTracer()),
-      rateLimitingMiddleware(getTracer(), logger, {
-        windowMs: 300000,
-        requestsUntilBlock: 200,
-      }),
+      rateLimitingMiddleware(getTracer(), logger, rateLimitinConfig),
```

Comment:
... so we don't have to repeated the same thing everywhere when we want to change one value.